doc/conf.py

```diff
@@ -0,0 +1,13 @@
+"""Sphinx configuration file for an LSST stack package.
+
+This configuration only affects single-package Sphinx documentation builds.
+"""
+
+from documenteer.conf.pipelinespkg import *  # noqa: F403, import *
```

Comment:
Since you have got docs you can in theory add the build_docs.yaml action from daf_butler.